app/models/enclosure_url_builder.rb

```diff
@@ -2,12 +2,32 @@
 require "addressable/template"
 
 class EnclosureUrlBuilder
+  def self.mark_authorized(enclosure_url, feed)
+    return enclosure_url unless feed.private?
+    raise "Missing tokens for private feed #{feed.id}" if feed.private? && !feed.tokens.any?
```

Comment:
This runtime error works as a contract with the apple publishing to halt the publishing flow if we don't have a way to pull down stitched audio.